src/MyShop.Api/MyShop.Api.csproj

```diff
@@ -15,6 +15,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<ProjectReference Include="..\..\SoapApi\SoapApi.csproj" />
```

Comment:
nie może być tego

Reply:
https://jason-ge.medium.com/create-and-consume-soap-based-service-in-net-6-4b6fc6161c8c